frontend/views.py

```diff
@@ -65,3 +70,40 @@ def get_page(request, file_name):
 
 def redirect_images_to_media(request, image):
     return redirect(settings.MEDIA_URL + "images/" + image)
+
+def submit_page_change(request):
+
+    if request.method == 'POST':
+
+        patch = get_diff(request.POST.get("page", ""),
+                         request.POST.get("content", ""))
+        form = PageEditForm({"page": request.POST.get("page", ""),
+                             "patch": patch,
+                             "comment": request.POST.get("descr", ""),
+                             "email": request.POST.get("email", ""),
+                             "date": datetime.datetime.now()})
+        if form.is_valid():
+            form.save()
+            return HttpResponse('')
+        else:
+            error_html = [ "<strong>{key}</strong>: {message}".format(key=key,
+                                                                      message=', '.join(message))
+                           for key, message in form.errors.items() ]
+            error_html = "<br>".join(error_html)
+            return HttpResponseForbidden(error_html)
+    else:
+        raise Http404
+
+
+def get_diff(page, content):
```

Comment:
We might want to use this lib? https://gitpython.readthedocs.io/en/stable/tutorial.html#obtaining-diff-information

I'm not sure